js/templates/modals/purchase/directPayment.html

```diff
@@ -0,0 +1,19 @@
+<% // duplicate the moderator card html to make sure everything aligns %>
```

Comment:
The direct payment button was broken out into it's own view and template so it's state could be updated without re-rendering the entire view. For example, it starts selected, but then switches to unselected when the first moderator loads.

Reply:
That’s such a crazy UX for me. You’re telling me the purchase screen loads with Direct selected and then some amount of time later, unbeknownst to the user, it changes?

I think that’s crazy. I might glance and see direct and  check out a couple other things, suddenly by the time I go to click purchase, thinking I’m making a direct order, I got me a moderator.

@morebrownies What do you think about this?

Reply:
Yeah I wouldn't recommending switching it on the user after more content loads. We can't guarantee they're going to see the selection change.

Either keep it as `direct` by default or don't select either option and let the user decide.

Reply:
or you select nothing and if nothing is still selected when the mod comes back, then select the mod.

If they click purchase while nothing is selected, we show an error forcing them to explicitly choose Direct or wait for the mods.

Reply:
I think it might be best to just simplify things and have either:
a. direct purchase is selected at first, and stays selected if the user doesn't change it once moderators have loaded. This is the simplest way to do it.

b. neither direct purchase nor any moderator is automatically selected, the user has to purposely select one before they can continue with their order. If no moderators that match the current criteria (coin, show only verified) have loaded, the user has to click on the direct payment option.

Reply:
We think alike @morebrownies - we were typing at the same time.

Reply:
Looks like we all basically said the same thing at the same time 🥇